.changeset/late-pugs-carry.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fuel-ts/interfaces": patch
+"@fuel-ts/address": patch
+"@fuel-ts/errors": patch
+---
+
+chore: deprecate bech32 addresses
```

Comment:
> [!Warning]
> Please do not merge this PR until further notice.

This is a hot topic, and there's been some last-minute pondering about this deprecation.



I'll reply to or close this comment with news once there is a consensus about it.

Reply:
Converting this to a draft until we have an consensus around this.